contracts/dev-utils/contracts/src/OrderTransferSimulationUtils.sol

```diff
@@ -134,19 +134,19 @@ contract OrderTransferSimulationUtils is
         // Transfer `makerAsset` from maker to taker
         assetData[1] = order.makerAssetData;
         fromAddresses[1] = order.makerAddress;
-        toAddresses[1] = takerAddress;
+        toAddresses[1] = takerAddress == address(0) ? address(1) : takerAddress;
```

Comment:
is this because some tokens prevent "burning" by sending to 0 address?

Reply:
Yeah, tokens typically `require(to != address(0x0))`. Since we're using the `OrderTransferSimulations` contract as part of validation now, we don't want the transfer to fail because the recipient is address zero (since a taker won't actually be address zero).